app/components/primer/octicon_component.rb

```diff
@@ -31,14 +31,16 @@ class OcticonComponent < Primer::Component
     #
     # @param icon [String] Name of <%= link_to_octicons %> to use.
     # @param size [Symbol] <%= one_of(Primer::OcticonComponent::SIZE_MAPPINGS) %>
+    # @param use_symbol [Boolean] Set to true when you setup a symbol dictionary using <%= link_to_component(Primer::OcticonSymbolComponent) %> on the page.
```

Comment:
Is `use_symbol` the best name? Could be `use:` or ??

Reply:
I think that the name is fine

Reply:
What is the long-term plan for this argument? Are we hoping to move _all_ implementations of Octicon to use this approach? I feel like it could be confusing to pass `true` here and not receive any error message if OcticonSymbol isn't then rendered on the page.